lib/util/ckan_auth_api.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from urllib.parse import urljoin
+
+import requests
+
+log = logging.getLogger(__name__)
+
+def get_jwt_from_ckan_authz(base_url, token, scope):
```

Comment:
Surely you want this as part of the private methods in your client? That way you don't need to pass `base_url` all over the place.

Reply:
And this applies for this whole file. I'd imagine these as private methods on your client lib.

Reply:
Is the `token` here the CKAN API key?

Did you consider using the base `ckanapi` library here (if so what were the pros/cons)?